shapash/webapp/smart_app.py

```diff
@@ -1564,12 +1543,13 @@ def update_feature_importance(label,
                                       apply_filters,
                                       reset_filter,
                                       nclicks_del,
-                                      is_open,
                                       n_clicks,
                                       bool_group,
                                       click_zoom,
                                       clickData,
```

Comment:
The arg clickData should be snake-cased click_data.
Also, can you remove unused args from all callbacks?